_episodes/04-function-style.md

```diff
@@ -367,8 +423,6 @@ The docstring should explain what the function or module does (and not how it is
 {: .callout}
 
 ### Sections of a Docstring
-There are many ways you could format this docstring (different styles/conventions). We recommend using [numpy style docstrings], and this is what the example above and `calculate_distance` function are written in.
-
 Each docstring has a number of sections which are separated by headings. Headings should be underlined with hyphens (`-----`). There are many options for sections, we will only cover the most relevant here. If you would like to see a full list, check out the documentation for [numpy style docstrings].
```

Comment:
Add line breaks please. Can we link to the NumPy explanation of docstrings - https://numpydoc.readthedocs.io/en/latest/format.html

The previous link no longer worked, but the above link is the updated reference.